pkgs/by-name/sn/snapcraft/package.nix

```diff
@@ -110,10 +101,11 @@ python3Packages.buildPythonApplication rec {
     validators
   ];
 
-  build-system = with python3Packages; [ setuptools ];
+  build-system = with python3Packages; [ setuptools-scm ];
```

Comment:
Just for my own info, what's the difference with `setuptools`?

Reply:
`setuptools-scm` can use the VCS system to automatically determine version (such as the git tag, for example)

Reply:
OK! Thanks!